lib/HLSL/HLOperationLower.cpp

```diff
@@ -6300,6 +6300,150 @@ Value *TranslateSelect(CallInst *CI, IntrinsicOp IOP, OP::OpCode opcode,
   }
   return Builder.CreateSelect(cond, t, f);
 }
+
+Value *TranslateMatVecMul(CallInst *CI, IntrinsicOp IOP, OP::OpCode opcode,
+                          HLOperationLowerHelper &helper,
+                          HLObjectOperationLowerHelper *pObjHelper,
+                          bool &Translated) {
+
+  hlsl::OP *hlslOP = &helper.hlslOP;
```

Comment:
My biggest immediate reaction here is that all new code contributed to DXC should conform to the LLVM coding standards. While we haven't always been great about enforcing that has always been the documented standard:

https://github.com/microsoft/DirectXShaderCompiler/blob/main/CONTRIBUTING.md#coding-guidelines

This all needs clang-format applied, and also should have all the variable naming updated to follow the standards:
https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
I tried to mimic the style of the file. Anyway updated to LLVM style.

Reply:
Resolved